ts/screens/modal/IdentificationModal.tsx

```diff
@@ -334,6 +343,32 @@ class IdentificationModal extends React.PureComponent<Props, State> {
     }
   };
 
+  private onLogout = () => {
+    Alert.alert(
+      this.props.profileName
```

Comment:
to avoid the repeat of the `profileName` check, what do you think to refactor this adding an utility function like this?

```typescript
  private loadAlertTexts = (
    profileName: string | undefined
  ): [string, string] =>
    fromNullable(profileName).fold(
      [
        I18n.t("identification.logout"),
        I18n.t("identification.logoutDescription")
      ],
      pn => [
        I18n.t("identification.logoutProfileName", {
          profileName: pn
        }),
        I18n.t("identification.logoutDescriptionProfileName", {
          profileName: pn
        })
      ]
    );
```

and then in the `Alert`:
```typescript
Alert.alert(
  ...this.loadAlertTexts(this.props.profileName,
  ...
)
```

could it be an improvement?